spec/models/customer_spec.rb

```diff
@@ -1,5 +1,38 @@
 require 'rails_helper'
 
 RSpec.describe Customer, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'should be possible login as a customer' do
+    customer = Customer.create!(email: 'john.doe@campuscode.com.br', password: '123456')
+    login_as customer
+    expect(customer.valid?).to eq(true)
+  end
+
+  it 'should block free emails' do
+    customer = Customer.create(email: 'john.doe@gmail.com', password: '123456')
+    login_as customer
+    expect(customer.valid?).to eq(false)
```

Comment:
Apesar disso ser comum no minitest porque a gente fazia `assert customer.valid?` ou `refure` no RSpec todo método que termina com `?` ele cria um [matcher de predicado](https://relishapp.com/rspec/rspec-expectations/v/3-10/docs/built-in-matchers/predicate-matchers) pra você então essa linha poderia virar:

```suggestion
    expect(customer).not_to be_valid
```

Mas acho que além de disso podia testar também que o erro está no local correto, o que acham?

Reply:
Esse comentário do matcher de predicado pode ser usado no arquivo todo

Reply:
Feito em todo arquivo de teste.